LottieSample/src/main/kotlin/com/airbnb/lottie/samples/PlayerFragment.kt

```diff
@@ -335,6 +340,17 @@ class PlayerFragment : Fragment() {
         updateUiFromState()
     }
 
+    private fun invertColor(color: Int) {
+        val shouldInvertColor = getContrastColor(color) == Color.WHITE
+        animationView.isActivated = shouldInvertColor
+        toolbar.isActivated = shouldInvertColor
+    }
+
+    private fun getContrastColor(color: Int): Int {
```

Comment:
You can make this a private extension function like:
```
@ColorInt
private fun Int.contrastColor(): Int { ... }
```

Reply:
It's also a bit strange that you return a color but then just check if the color is white to create shouldInvertColor.

You could rename it `Int.requiresLightForeground()` or `Int.isDark()` or something